crates/pop-cli/src/commands/up/contract.rs

```diff
@@ -54,7 +54,23 @@ pub struct UpContractCommand {
 impl UpContractCommand {
 	pub(crate) async fn execute(&self) -> anyhow::Result<()> {
 		clear_screen()?;
+
+		// Check if build exists in the specified "Contract build folder"
+		let build_path = PathBuf::from(
+			self.path.clone().unwrap_or("/.".into()).to_string_lossy().to_string()
+				+ "/target/release/ink",
+		);
+
+		if !build_path.exists() {
+			intro(format!("{}: Building a contract", style(" Pop CLI ").black().on_magenta()))?;
```

Comment:
A `warning("NOTE: contract has not yet been built")` here before building would be nice. :)

Reply:
Oh, sounds good.

I don't understand the intent, should it be used alongside the current intro ("building a contract") or replace this altogether?

Reply:
Alright, that makes sense!